documentation/modules/exploit/unix/webapp/opensis_chain_exec.md

```diff
@@ -0,0 +1,56 @@
+## Vulnerable Application
+
+This module exploits multiple vulnerabilities in [openSIS](https://www.opensis.com/) 7.4 and prior versions which could be abused by unauthenticated attackers to execute arbitrary PHP code. It is based on these advisories:
+
+- http://karmainsecurity.com/KIS-2020-06
+- http://karmainsecurity.com/KIS-2020-07
+- http://karmainsecurity.com/KIS-2020-08
+
+The module has been successfully tested against [openSIS](https://www.opensis.com/) versions 7.3 and 7.4 running on Ubuntu. Older versions might be affected as well.
```

Comment:
I presume this was tested on the community edition. If so, please add a link to download the software here somewhere.

* https://sourceforge.net/projects/opensis-ce